packages/devtools_app/lib/src/shared/config_specific/import_export/_export_desktop.dart

```diff
@@ -21,4 +23,9 @@ class ExportControllerDesktop extends ExportController {
   }) {
     _fs.writeStringToFile(fileName, content);
   }
+
+  @override
+  void saveDataFile({required Uint8List content, required String fileName}) {
+    throw UnimplementedError();
```

Comment:
can you write this to file (like above) so that we can still use this feature on the Desktop build of DevTools?

Reply:
YEs, this is the plan, but can i do it is separate PR. I want to squise the fix to release.

Reply:
actually, if you merge to use one method (see my comment below), then you can use the existing override above.

Reply:
May be. I need to test it.